test/msw-api/handle-stream.mocks.ts

```diff
@@ -0,0 +1,10 @@
+import { setupWorker } from 'msw'
+
+const worker = setupWorker()
+
+worker.on('response:bypass', async (res) => {
+  const textResponse = await res.text()
```

Comment:
This setup (and its test) are relevant in the context of life-cycle events. Maybe let's move them both into `test/msw-api/life-cycle-events`? 

Ideally, it would be nice to introduce this test in `test/rest-api/response/body/stream` and add an assertion that MSW is able to read that stream (which I think you already have).

Reply:
The behavior is a regression of `life-cycle` apis. So as you said I think that is better to go there. 

Do you prefer to have in `test/rest-api/response/body/stream` or `test/msw-api/life-cycle-events` ?

Reply:
I think there's little difference, to be honest. As it's a regression, let's keep it close to the problematic surface, that being life-cycle events API.

Reply:
I agree with you 👍